src/drtsans/filter_events.py

```diff
@@ -0,0 +1,412 @@
+from operator import itemgetter
+from typing import List, Optional, Tuple
+
+from mantid.api import AnalysisDataService, WorkspaceGroup
+from mantid.dataobjects import EventWorkspace
+from mantid.simpleapi import (
+    AddSampleLog,
+    CreateEmptyTableWorkspace,
+    FilterEvents,
+    GenerateEventsFilter,
+    GroupWorkspaces,
+    LoadEventNexus,
+    logger,
+    mtd,
+)
+
+from drtsans import PV_POLARIZER_FLIPPER, PV_ANALYZER_FLIPPER
+from drtsans.polarization import PV_POLARIZER, PV_POLARIZER_VETO, PV_ANALYZER, PV_ANALYZER_VETO
+from drtsans.samplelogs import SampleLogs
+from drtsans.type_hints import MantidWorkspace
+
+
+def workspace_handle(input_workspace: MantidWorkspace):
+    """Syntactic sugar for a more descriptive operation"""
+    return mtd[str(input_workspace)]
+
+
+def extract_times(
```

Comment:
can you move this and the other functions to module polarization.py? `filter_events` is too broad

Reply:
I renamed to `polarization_filter_events.py`, to keep files smaller and in case we end up deleting the whole module. Does that sound okay?